src/p_mobj.c

```diff
@@ -58,6 +58,16 @@ void P_UpdateDirectVerticalAiming(void)
   max_pitch_angle = default_max_pitch_angle * ANG1;
 }
 
+void P_SetActualHeight(mobj_t *mobj)
+{
+    spritedef_t *sprdef = &sprites[mobj->sprite];
+    spriteframe_t *sprframe = &sprdef->spriteframes[mobj->frame & FF_FRAMEMASK];
+    if (sprframe)
+    {
+        mobj->actualheight = sprframe->lump[0];
```

Comment:
You're assigning a lump number, not the height.

Reply:
Right, fixed here: <https://github.com/fabiangreffrath/woof/pull/2331/commits/be0fdd129180dc28af9cf8e288d3feaa44c4c00f> Thanks.